### ISSUE TITLE
FIND_TUNNEL list full of bad nodes

### ISSUE THREAD

**Issue by littleskunk** [Storj/core#186]
### Package Versions

Replace the values below using the output from `npm list storj`.

```
/usr/local/lib
└─┬ storjshare-cli@4.0.3
  └─┬ storj@1.2.7
    └── kad@1.5.13
```

Replace the values below using the output from `node --version`.

```
v4.4.5
```

### Expected Behavior

Please describe the program's expected behavior. Include an example of your
usage code in the back ticks below if applicable.

FIND_NODE sould return a list of working tunnel gateway nodes. If the firt node is not working the second or third should be fine.

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

My node is sending 19 OPEN_TUNNEL. Only one node send me an answer but 2 seconds to late. All other nodes timed out. I see three possible reason for this.
1.) A few nodes have problems with signature expired cause of to many pending offers. https://github.com/Storj/core/issues/185
2.) Port forward rule for the farmer port but not for the tunnel gateway ports.
3.) Node is offline.

1 and 2 will send out tunnel PUBLISH messages every 15 minutes but they are not able to open a tunnel connection.

Every time a node needs a tunnel it will request the tunnel list and use one connection. Sooner or later all good nodes are removed and only the bad nodes are remaining. Now we have a FIND_TUNNEL list full of bad nodes and nobody can open a new tunnel connection.

My log 
[storj.txt](https://github.com/Storj/core/files/321305/storj.txt)

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Start a node that needs a tunnel connection

**Comment by bookchin** [comment]
I think we can rule out 2, as we just recently issued a fix for that and the test suite indicates that it is working properly. I tend to lean towards 1. 3 is possible, but I think less likely than 1.

**Comment by bookchin** [comment]
This will get closed with #195, but keep let's keep an eye on it and reopen if the changes there do not fully resolve.

**Comment by littleskunk** [comment]
Roger that.
